### ISSUE TITLE
[Mod Compatibility]: Plants 2 Nether tree configs

### ISSUE THREAD

**Issue by TeetoKaziim** [Shovinus/ChopDownUpdated#30]
https://gist.github.com/TeetoKaziim/1360fabb5772e11c503dde94a73f6c4c

I split up the gist into seperate files, to ease use. Sorry for missing these the first time.

**Comment by Shovinus** [comment]
Is this trees I have missed from last version of the mod?

**Comment by TeetoKaziim** [comment]
This is specifically the uh, nether trees from plants2. 

I somehow neglected to include them in the config examples I sent you in the issue I made for plants2, the first time, so I'm sending them now. Especially with the change to the config where user-added trees are no-longer-a-thing, I figured you would want all the trees for the mod.

In released versions of the mod, these trees do not presently fall over when whacked, while all-other plants2 trees do.

Sorry about that, totally my fault.

![image](https://user-images.githubusercontent.com/25490944/49193437-aec0d980-f344-11e8-87ba-f008dd799cdf.png)
Pictured here is me having knocked over one of the trees in question using the custom config submitted in the gist, in the ``plants.txt`` section, on the 1.2.0 version of chopdown updated.
